core/commands/repo.go

```diff
@@ -39,6 +40,12 @@ var RepoCmd = &cmds.Command{
 	},
 }
 
+// GcResult is the result returned by "repo gc" command
+type GcResult struct {
```

Comment:
We can actually remove this whole type in favor of the one in the gc package (just change the field names so the json marshaling matches up correctly)